cli/runtime/config/clientconfignextgen_filesystem.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
question: what's the rationale for naming a set of files in this dir **_filesystem**.go? 
maybe _file.go would be more appropriate? 
(If we elect to change, and especially since there are preexisting files already committed, let's do it as a followup.)